bom/pom.xml

```diff
@@ -66,7 +66,7 @@
         <plexus-component-annotations.version>1.7.1</plexus-component-annotations.version>
         <!-- What we actually depend on for the annotations, as latest Graal
            is not available in Maven fast enough: -->
-        <graal-sdk.version>1.0.0-rc16</graal-sdk.version>
+        <graal-sdk.version>19.0.0</graal-sdk.version>
```

Comment:
Shouldn't it be `19.0.2`?

Reply:
Those artifacts were not available last time I checked.

Reply:
I think it's there now: https://search.maven.org/artifact/org.graalvm.sdk/graal-sdk/19.0.2/jar ? Or is it a different one?

Reply:
I can give it a try again.

Reply:
Sure enough, seems to work.  Pushed